fsharp-backend/src/LibBackend/UserDB.fs

```diff
@@ -1,7 +1,17 @@
+/// <summary>
+/// Supports anything relating to Datastores on a user canvas.
+/// Namely, this file is responsible for managing this user data, including CRUD and type checking.
+/// </summary>
+///
+/// <remarks>
+/// User data stores live within the `user_data` table, which is broken down by:
+/// - Dark version
+/// - canvas (by ID)
+/// - table/store ID
+/// - user version (version of the DB - incremented by any structural change to the store)
```

Comment:
This isn't quite right. User version is for when the users change the type being stored. We also have a dark version tag which is for if we decide to store the data differently.